WhatsApp/app/src/main/res/layout/fragment_home.xml

```diff
@@ -85,7 +80,7 @@
 
 
         <com.google.android.material.floatingactionbutton.FloatingActionButton
-            android:id="@+id/fabNewConversation"
+            android:id="@+id/fabbtn"
```

Comment:
why did we rename this id

Reply:
felt like it was too long tried to shorten it